packages/@aws-cdk/aws-iam/lib/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { Resource } from '@aws-cdk/core';
+import { Construct, IConstruct } from 'constructs';
+import { IPrincipal } from './principals';
+
+/**
+ * Determines whether the given Principal is a newly created resource managed by the CDK,
+ * or if it's only an imported existing resource.
```

Comment:
"referenced" is the new term, "importing" is confusing

Reply:
Fixed.